pkg/base1/test-spawn-proc.js

```diff
@@ -91,6 +91,32 @@ QUnit.test("error message fail", function (assert) {
             });
 });
 
+QUnit.test("nonexisting executable", function (assert) {
+    const done = assert.async();
+    assert.expect(2);
+    cockpit.spawn(["/bin/nonexistent"])
```

Comment:
Please rewrite this using async/await syntax.

Reply:
I think this is actually best expressed with `assert.rejects`. WDYT?

Reply:
I agree.